[ONNX] Skip doctest `torch.onnx._internal.fx` if ImportError

BowenBao opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #95935
* #95929
* #95676
* #95664
* #95651
* __->__ #95686
* #95650

Need to use `exclude` to skip the module altogether. Because xdoctest
triggers `ImportError` when trying to import the module. So the whole
test fails regardless if skip was added in the docstring.

BowenBao committed:
skip doctest onnx internal

[ghstack-poisoned]

BowenBao committed:
Update on "[ONNX] Skip doctest `torch.onnx._internal.fx` if ImportError"


Need to use `exclude` to skip the module altogether. Because xdoctest
triggers `ImportError` when trying to import the module. So the whole
test fails regardless if skip was added in the docstring.

[ghstack-poisoned]

BowenBao committed:
Update on "[ONNX] Skip doctest `torch.onnx._internal.fx` if ImportError"


Need to use `exclude` to skip the module altogether. Because xdoctest
triggers `ImportError` when trying to import the module. So the whole
test fails regardless if skip was added in the docstring.

[ghstack-poisoned]

BowenBao committed:
Update on "[ONNX] Skip doctest `torch.onnx._internal.fx` if ImportError"


Need to use `exclude` to skip the module altogether. Because xdoctest
triggers `ImportError` when trying to import the module. So the whole
test fails regardless if skip was added in the docstring.

[ghstack-poisoned]

BowenBao committed:
Update on "[ONNX] Skip doctest `torch.onnx._internal.fx` if ImportError"


Need to use `exclude` to skip the module altogether. Because xdoctest
triggers `ImportError` when trying to import the module. So the whole
test fails regardless if skip was added in the docstring.

[ghstack-poisoned]

kit1980 commented on test/run_test.py:
```diff
@@ -709,6 +711,17 @@ def run_doctests(test_module, test_directory, options):
+            import onnxscript  # NOQA
+            import onnxruntime  # NOQA
+        except ImportError:
+            exclude_module_list += ["torch.onnx._internal.fx.*"]
```
`exclude_module_list.append("torch.onnx._internal.fx.*")` ?

BowenBao committed:
Update on "[ONNX] Skip doctest `torch.onnx._internal.fx` if ImportError"


Need to use `exclude` to skip the module altogether. Because xdoctest
triggers `ImportError` when trying to import the module. So the whole
test fails regardless if skip was added in the docstring.

[ghstack-poisoned]

BowenBao commented:
@pytorchbot merge -g

BowenBao committed:
Update on "[ONNX] Skip doctest `torch.onnx._internal.fx` if ImportError"


Need to use `exclude` to skip the module altogether. Because xdoctest
triggers `ImportError` when trying to import the module. So the whole
test fails regardless if skip was added in the docstring.

[ghstack-poisoned]

BowenBao commented:
@pytorchbot merge -g
